.github/workflows/main.yaml

```diff
@@ -71,16 +74,17 @@ jobs:
   style:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
+    - name: Checkout
+      uses: actions/checkout@v4
 
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v5
       with:
-        python-version: '3.11'
+        python-version: 3.11
```

Comment:
I believe this was put in quotes when using `3.10` because as a number that gets converted to `3.1`, so could keep it to be safe but should be fine in this case

Reply:
Yeah I removed because on publish was not using quotes but as you said since is 3.11 will not cause warm.